resources/js/pages/forums/recent-posts.tsx

```diff
@@ -8,13 +8,14 @@ import { RecentPostsMainRoot } from '@/features/forums/components/RecentPostsMai
 const RecentPosts: AppPage = () => {
   const { t } = useTranslation();
 
+  const metaDescription =
+    'A list of recent posts that have been made on the RetroAchievements forum.';
+
   return (
     <>
       <Head title={t('Recent Posts')}>
-        <meta
-          name="description"
-          content="A list of recent posts that have been made on the RetroAchievements forum."
-        />
+        <meta name="description" content={metaDescription} />
+        <meta name="og:description" content={metaDescription} />
```

Comment:
I'm not seeing these on any of the pages. Here's what I see for Recent Posts:
```
<html>
<body>
<!--StartFragment-->
<link rel="icon" type="image/png" href="http://localhost:64000/assets/images/favicon-gray.webp">
--
  | <link rel="preload" as="image" importance="high" href="http://localhost:64000/assets/images/ra-icon.webp">
  | <link rel="image_src" href="http://localhost:64000/assets/images/ra-icon.webp">
  | 
  | <meta name="copyright" content="Copyright 2014-2024">
  | <meta name="keywords" content="games, achievements, retro, emulator">
  | <meta name="format-detection" content="telephone=no">
  | <meta property="og:title" content="RetroAchievements">
  | <meta property="og:image" content="http://localhost:64000/assets/images/favicon.webp">
  | <meta property="og:url" content="http://localhost:64000/forums/recent-posts">
  | <meta property="og:type" content="website">
  | <meta name="theme-color" content="#161616">
  | <link rel="canonical" href="http://localhost:64000/forums/recent-posts">
  | <link rel="preconnect" href="https://media.retroachievements.org">
  | <link rel="dns-prefetch" href="https://media.retroachievements.org">
  | <link rel="preconnect" href="">
  | <link rel="dns-prefetch" href="">

<!--EndFragment-->
</body>
</html><link rel="icon" type="image/png" href="http://localhost:64000/assets/images/favicon-gray.webp">
    <link rel="preload" as="image" importance="high" href="http://localhost:64000/assets/images/ra-icon.webp">
    <link rel="image_src" href="http://localhost:64000/assets/images/ra-icon.webp">
    <meta name="csrf-token" content="Cxijtu1IHa9R48h3xagXL8ry7m5jjiRYxw73ijjH">
    <meta name="copyright" content="Copyright 2014-2024">
    <meta name="keywords" content="games, achievements, retro, emulator">
    <meta name="format-detection" content="telephone=no">
            <meta property="og:title" content="RetroAchievements">
    <meta property="og:image" content="http://localhost:64000/assets/images/favicon.webp">
    <meta property="og:url" content="http://localhost:64000/forums/recent-posts">
    <meta property="og:type" content="website">
    <meta name="theme-color" content="#161616">
    <link rel="canonical" href="http://localhost:64000/forums/recent-posts">
    <link rel="preconnect" href="https://media.retroachievements.org/">
    <link rel="dns-prefetch" href="https://media.retroachievements.org/">
    <link rel="preconnect" href="">
    <link rel="dns-prefetch" href="">
    ```

Reply:
I see them, but it's near the end of the `<head>` tag:

<img width="952" alt="Screenshot 2024-12-06 at 9 47 08 PM" src="https://github.com/user-attachments/assets/44343bcf-d9da-4e11-9fee-6ca5cf6c59e3">

<img width="818" alt="Screenshot 2024-12-06 at 9 46 42 PM" src="https://github.com/user-attachments/assets/fac2f3f4-4b0a-4b9f-902f-998a49cb85f4">

In your browser devtools, in the "Elements" tab, you should be able to search for "og:description" and find the elements.

Reply:
Ah, this will happen when not running SSR locally.

Try `sail pnpm build` and then `sail artisan inertia:start-ssr`:
![Screenshot 2024-12-10 at 9 24 32 AM](https://github.com/user-attachments/assets/30b3086b-8c60-4b72-9711-fd2ef0a76321)